Engine/DataFeeds/ZipDataCacheProvider.cs

```diff
@@ -57,6 +58,9 @@ public ZipDataCacheProvider(IDataProvider dataProvider, bool isDataEphemeral = t
         /// </summary>
         public Stream Fetch(string key)
         {
+            // Will throw while trying to read ZIP file if this line is not included
+            Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
```

Comment:
link to stackoverflow issue

Reply:
Might also want to move this registration to a static constructor or somewhere else that ensure we only run it once per `AppDomain`